pkg/git/git.go

```diff
@@ -57,108 +54,74 @@ var preReceiveHookTpl = template.Must(template.New("hooks").Parse(preReceiveHook
 //
 // Returns:
 // 	- nothing
```

Comment:
@smothiki can you remove the `Params` and `Returns` sections - we no longer need them since they're shown in the function definition